logic/account.js

```diff
@@ -580,13 +582,33 @@ Account.prototype.getAll = function (filter, fields, cb) {
 	delete filter.offset;
 
 	if (filter.sort) {
-		sort = filter.sort;
+		sort = orderBy.sortQueryToJsonSqlFormat(filter.sort, ['username', 'balance']);
 	}
 	delete filter.sort;
 
-	if (typeof filter.address === 'string') {
-		filter.address = {
-			$upper: ['address', filter.address]
+	if (filter.address) {
+
+		if (typeof filter.address === 'string') {
+			filter['a.address'] = {
+				$upper: ['a.address', filter.address]
+			};
+		}
+
+		if (typeof filter.address === 'object') {
```

Comment:
In which form are you expecting the address to be? I think the better idea is to accept it in the same way or at least to have some better validation

Reply:
Right, I'll update it.